Moonrake.sln

```diff
@@ -9,6 +9,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GameEngine", "GameEngine\Ga
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GameData", "GameData\GameData.csproj", "{7E387CF0-17BD-42C1-9DF4-B208153DDE4B}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SoundUtils", "SoundUtils\SoundUtils.csproj", "{5BB92D21-99F0-4C40-BD84-63E7F5391716}"
+EndProject
```

Comment:
This change has already been added to master. This pull request shouldn't be adding it again.

Reply:
Can you create a new branch and create your pull request from there instead of trying to re-use this old dev branch?

Reply:
ok